v20.2/drop-role.md

```diff
@@ -14,6 +14,7 @@ The `DROP ROLE` [statement](sql-statements.html) removes one or more SQL roles.
 
 - The `admin` role cannot be dropped, and `root` must always be a member of `admin`.
 - A role cannot be dropped if it has privileges. Use [`REVOKE`](revoke.html) to remove privileges.
+- Roles that [own objects](authorization.html#object-ownership) cannot be dropped until the [ownership is transferred to another role](owner-to.html#change-a-databases-owner).
```

Comment:
Suggest adding "objects such as databases, tables ..." since someone who is familiar with SQL but not necessarily with CockroachDB could reasonably wonder what are "objects" referring to.